content/root/quick-start-guide.textile

```diff
@@ -15,65 +17,84 @@ languages:
   - flutter
 jump_to:
   Help with:
-    - Adding the Ably Library#adding-library
-    - Receiving Messages#receiving-messages
-    - Sending Messages#sending-messages
+    - Add the Ably Client Library SDK#step-1
+    - Connect to Ably#step-2
+    - Subscribe to a channel#step-3
+    - Publish a message#step-4
+    - Close a connection to Ably#step-5
 ---
 
-Integrating Ably is simple.  In this guide we'll run you through the basics.
+Welcome to Ably. This guide shows you how to integrate Ably's realtime platform into an application.
 
-Before we dive into code, there are few concepts you should understand:
+You'll learn how to:
 
-- Messages carry data := Data published through the Ably platform is packaged in a message. Each message contains, amongst other things, an optional event name, data payload and an Ably generated unique ID.
-- Channels for distribution := All message traffic is organized by user-specified named channels within the Ably service. These channels are the "unit" of message distribution; clients attach to channels to subscribe to messages, and every message published to a channel is broadcast by Ably to all subscribers. This scalable messaging pattern is commonly called "pub/sub":https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern.
-- Realtime library := Ably's realtime client libraries are most commonly used client-side on mobile and web devices. They establish a persistent multiplexed bi-directional socket connection to Ably, which in turn, allows Ably to push messages and events to clients. Realtime libraries maintain their connection, channels and presence state. <span lang="ruby">When using the Realtime library in Ruby, you need to run it within an "Event Machine":https://github.com/eventmachine/eventmachine. "Find out more about using the Realtime library":/documentation/realtime/usage.</span>
-- REST library := Ably's REST libraries are most commonly used server-side and communicate with Ably over HTTP. As they are HTTP based, they cannot subscribe to messages in real time, however they can publish messages, issue tokens, retrieve history and perform all other operations available via the "Ably REST API":/rest-api/.
+* Include an Ably Client Library SDK in a project.
+* Establish a realtime connection to Ably.
+* Subscribe to a channel to receive messages.
+* Publish a message on a channel.
+* Close a connection to Ably.
 
-h2(#adding-library). Adding the Ably library
+Ably Client Library SDKs are available in multiple programming languages. Some SDKs have a realtime and a REST interface, whilst others only have a REST interface. The "SDK downloads":https://ably.com/download page lists availability. For further information on when it's best to use the realtime or REST SDK, see "the best practice guide":https://ably.com/documentation/best-practice-guide#choosing-between-realtime-rest-mqtt.
```

Comment:
Would suggest using 'while' here instead of 'whilst', for the benefit of international readers.

Reply:
Updated.